lib/napa/grape_extenders.rb

```diff
@@ -7,12 +7,12 @@ def self.extended(modified_class)
       if defined?(::ActiveRecord)
         modified_class.rescue_from ::ActiveRecord::RecordNotFound do |e|
           err = Napa::JsonError.new(:record_not_found, 'record not found')
-          Napa::Logger.logger.debug Napa::Logger.response(404, {}, err)
+          Napa::Logger.logger.debug Napa::Logger.response_format(404, {}, err)
```

Comment:
Good call! I've got a change that I'll push up shortly that wraps these in helper methods.